docs/guides/platform/object-storage/using-object-storage-with-akamai-cdn/index.md

```diff
@@ -0,0 +1,101 @@
+
+The solution presented in this guide focuses on the architectural best practices for designing and operating reliable, secure, efficient, and sustainable content storage and delivery systems using [Linode Object Storage](https://techdocs.akamai.com/cloud-computing/docs/object-storage) on Akamai Connected Cloud with [Akamai CDN](https://techdocs.akamai.com/platform-basics/docs/content-delivery).
+
+## Object Storage For Data Delivery
+
+With the right [bucket architecture](/docs/guides/optimizing-obj-bucket-architecture-for-akamai-cdn/), object storage can be used to house content for effective unstructured data delivery. Object storage supports file-critical features such as encryption, compression, deduplication, and versioning, and its accessibility via HTTP protocols offers instant access to objects.
+
+For example, these factors make object storage ideal for storing unstructured data like video and audio files that don’t require frequent updating and services that require efficient input/output.
+
+## Object Storage Specifications, Considerations, and Strategies for Streaming Data
+
+### Technical Specifications
+
+When considering applications -- like streaming video or audio -- that involve sourcing data from object storage, there are a few things to keep in mind, including content access, bandwidth limitations, latency sensitivity, scalability, egress costs, and other technical limitations:
+
+-   **Bandwidth and latency:** Object storage solutions are not always optimized for high-bandwidth streaming applications. This can lead to potential buffering issues and slower delivery speeds.
+
+-   **Scalability:** To preserve infrastructure integrity, object storage systems may have limitations in place for concurrent connections and simultaneous user streams. This can impact the ability to efficiently handle spikes in demand.
+
+-   **Retrieval speed:** Without CDN edge-based caching, content streamed directly from object storage origin servers may result in slower data retrieval speeds and higher latency.
+
+-   **Data transfer and egress costs:** Streaming directly from object storage can incur high egress costs since each user request may result in data transfer fees, whereas CDNs cache data at the edge to optimize delivery.
+
+-   **Resource limits:** Object storage systems may have limitations on the size of individual objects, potentially restricting the size of videos that can be efficiently streamed.
+
+-   **Rate limits:** Object storage solutions often have rate limits imposed on a requests per second (RPS) basis. If users are accessing Object Storage endpoints directly without the use of a CDN, there is a greater chance RPS limits are reached as more users access content.
+
+-   **Updating manifests:** Bucket contents can be determined by syncing with a manifest. This means that when source manifests are updated often, buckets may constantly have their contents changed.
+
+### Internet Considerations
```

Comment:
Maybe a fancy phrase like Network Topology could be a good heading here (I haven't done any SEO research on that, but my mind was reaching for other ways to state this)

Reply:
Also, I think this section would *eventually* be better served as a full guide explaining these concepts. We're making some assertions about the concepts in this section that feel like they deserve more explanation

Reply:
Network topology is interesting, but I think that's more specific to node-based architecture within a system, rather than general. Renamed to "Network considerations" since "Internet" sounded weird. 

Agree we could use content fleshing out the explanations here.